sycl/test-e2e/Graph/device_query.cpp

```diff
@@ -31,3 +29,11 @@ int main() {
     assert(SupportsGraphs == exp_ext::graph_support_level::unsupported);
   }
 }
+
+int main() {
+  queue Queue;
+  test(Queue);
+
+  queue ioq{sycl::gpu_selector_v, {sycl::property::queue::in_order()}};
```

Comment:
`sycl::gpu_selector_v` seems like it will prohibit CPU devices, for example OpenCL CPU which can be used

Reply:
sycl::gpu_selector_v is removed